tests/fixtures/integration/actions/config/test_direct_interactive_ee.py/test/2.json

```diff
@@ -3,30 +3,10 @@
     "index": 2,
     "comment": "cache plugin details",
     "additional_information": {
-        "present": [],
+        "present": [
+            "option: CACHE_PLUGIN_TIMEOUT"
+        ],
         "absent": [],
-        "compared_fixture": true
-    },
-    "output": [
-        "Cache plugin timeout (current: 42)  (default: 86400)",
-        " 0│---",
-        " 1│current_config_file: None",
-        " 2│current_value: 42",
-        " 3│default: false",
-        " 4│default_value: 86400",
-        " 5│description: Expiration timeout for the cache plugin data.",
-        " 6│env:",
-        " 7│- name: ANSIBLE_CACHE_PLUGIN_TIMEOUT",
-        " 8│ini:",
-        " 9│- key: fact_caching_timeout",
-        "10│  section: defaults",
-        "11│name: Cache plugin timeout",
-        "12│option: CACHE_PLUGIN_TIMEOUT",
-        "13│source: env",
-        "14│type: integer",
-        "15│via: ANSIBLE_CACHE_PLUGIN_TIMEOUT",
-        "16│yaml:",
-        "17│  key: facts.cache.timeout",
-        "^b/PgUp page up                          ^f/PgDn page down                          ↑↓ scroll                          esc back                          - previous                          + next                          [0-9] goto                          :help help"
-    ]
+        "compared_fixture": false
+    }
```

Comment:
The output is programatically added to every fixture:

https://github.com/ansible/ansible-navigator/blob/62cf127289fe85df9b69b7cd10f4914c19b81bdc/tests/integration/_common.py#L109

So there's not really any reason to remove it, next time the fixtures are updated it will just come back.

Reply:
I will fix this in a different PR, avoiding to generate fixtures for entries with `compared_fixture=false`, we have good reasons for these as generating fixtures generates different results on different machines.